src/ebddocx2table/docxtableconverter.py

```diff
@@ -263,6 +266,74 @@ def _handle_single_table(
                     )
                 )
 
+    # see above boolean_outcome and subsequent_step_number could be ignored iff schemes of *-numbers are always the same
+    # pylint:disable=too-many-locals
+    def _handle_single_table_star_exception(
+        self,
+        table: Table,
+        multi_step_instructions: List[MultiStepInstruction],
+        row_offset: int,
+        rows: List[EbdTableRow],
+        row_index: int,
+    ) -> None:
+        """
+        Completes table when handling of single table (out of possible multiple tables for 1 EBD) hit a step
+        with several instructions. Those instructions will be split in individual steps.
+        As above, the results are written into rows, sub_rows and multi_step_instructions. Those will be modified.
+        """
+        use_cases: List[str] = []
+        # first init
+        enhanced_table_row = self._enhance_list_view(table=table, row_offset=row_offset)[row_index]
```

Comment:
magst du das `self._enhance_list_view(table=table, row_offset=row_offset)` noch in eine variable mit lesbasrem namen verpacken? 
du verwendest es ja an 4 stellen im nachfolgenden.

Reply:
1b287fa6e6e82b73ae5c37d4aebee3373b58f4a8